pkg/cdc/generate_config.go

```diff
@@ -91,8 +91,11 @@ func updateAdditionalJVMOpts(optsSlice []string, CDCConfig *cassdcapi.CDCConfigu
 		}
 	}
 	CDCOpt := fmt.Sprintf("-javaagent:%s=%s", "/opt/cdc_agent/cdc-agent.jar", strings.Join(optsSlice, ","))
+	// We want to disable MCAC when the server being deployed is DSE
+	if cassDC.Spec.ServerType == "cassandra" {
+		out = append(out, "-javaagent:/opt/metrics-collector/lib/datastax-mcac-agent.jar")
+	}
 	out = append(out, // We need to add these two elements to the slice first, because the management agent must start before the CDC agent.
-		"-javaagent:/opt/metrics-collector/lib/datastax-mcac-agent.jar",
 		"-javaagent:/opt/management-api/datastax-mgmtapi-agent-0.1.0-SNAPSHOT.jar",
```

Comment:
With the latest Management API images, the MAAC agent jarfile can be specified as simply:
```
/opt/management-api/datastax-mgmtapi-agent.jar
```

Reply:
Oh sorry, I forgot that bit! Fixing.